src/main/java/org/sonatype/gradle/plugins/scan/ossindex/OssIndexAuditTask.java

```diff
@@ -90,6 +92,12 @@ public void audit() {
         response = ossIndexClient.requestComponentReports(packageUrls);
       }
 
+      Set<String> vulnerabilityIds = extension.getExcludeVulnerabilityIds();
+      Set<PackageUrl> coordinates = toPackageUrls(extension.getExcludeCoordinates());
```

Comment:
Maybe `coordinates` to `coordinatesToExclude`?

Reply:
> Maybe `coordinates` to `coordinatesToExclude`?

👍 fc1fe2a5